### PR TITLE
Add reply functionality

### PR DESCRIPTION
**Pull request by HacksForHugs**
Fix: #128

### CODE REVIEW

File: app/controllers/replies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  private
+
+    # TODO (Shimmy): Add picture support
+  def reply_params
+    params.require(:reply).permit(:content, :post_id, :user_id).merge(user_id: current_user.id)
+  end
+
+  # TODO (Shimmy): Use CanCanCan instead.
```
Comment (lilshim) [inline app/controllers/replies_controller.rb]: delete if you're not using

File: app/models/post.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ class Post < ActiveRecord::Base
 
   # This will allow order_by_created_at to work, as well as order_by_created_at(:asc)
   scope :order_by_created_at, -> (direction = :desc) { order(created_at: direction) }
-
+  has_many :replies, dependent: :destroy
```
Comment (hidesminimally) [inline app/models/post.rb:7]: nice

File: app/views/groups/show.html.erb [hunk 3]
```diff
@@ -23,8 +23,9 @@
     <%= render 'groups/info_box' %>
 </div>
 <div class="row">
-  <div class="large-7 column">
-    <%= render 'posts/index', posts: @posts %>
+  <div class="group-posts large-7 column">
+
```
Comment (hidesminimally) [inline app/views/groups/show.html.erb:27]: delete extra space

File: app/views/replies/_new.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<div class="new-reply large-12 columns">
+  <%= form_for [post, @reply], remote: true do |f| %>
+    <div class="field">
+      <%= f.text_area :content, placeholder: "Write your response here" %>
+    </div>
+    <%= f.hidden_field :post, value: post, required: true %>
+    <%= f.submit "Post Reply Here", class: "button right", require: %>
```
Comment (hidesminimally) [inline app/views/replies/_new.html.erb]: delete the require aprt here

File: app/views/replies/_new.html.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<div class="new-reply large-12 columns">
+  <%= form_for [post, @reply], remote: true do |f| %>
+    <div class="field">
+      <%= f.text_area :content, placeholder: "Write your response here" %>
```
Comment (hidesminimally) [inline app/views/replies/_new.html.erb]: make this required

**Pull request merged**